eng/packaging.targets

```diff
@@ -42,4 +51,63 @@
     <MSBuild Projects="$(PkgProjPath)"
              Targets="Build" />
   </Target>
+    
+  <Target Name="AddRefAssemblies" Condition="'$(ExcludeFromPackage)' == '' and '$(TargetsAnyOS)' == 'true'"> 
+    <PropertyGroup>
+      <RefPath>$(BaseOutputPath)/ref/$(TargetFrameworkWithoutSuffix)-$(Configuration)/$(TargetName).dll</RefPath>
+    </PropertyGroup>
+    
+    <ItemGroup Condition="Exists('$(RefPath)')">
+      <TfmSpecificPackageFile Include="$(RefPath)">
+        <PackagePath>ref/$(TargetFrameworkWithoutSuffix)</PackagePath>
+      </TfmSpecificPackageFile>
+      <TfmSpecificPackageFile Include="$(XmlDocFileRoot)1033\$(TargetName).xml" Condition="Exists('$(XmlDocFileRoot)1033\$(TargetName).xml')">
+        <PackagePath>ref/$(TargetFrameworkWithoutSuffix)</PackagePath>
+      </TfmSpecificPackageFile>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="AddRuntimeSpecificAssemblies"
+          DependsOnTargets="BuiltProjectOutputGroup;SatelliteDllsProjectOutputGroup;_AddPriFileToPackBuildOutput;$(TargetsForTfmSpecificBuildOutput)"
+          Condition="'$(TargetsAnyOS)' != 'true' and '$(ExcludeFromPackage)' == ''">
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="@(SatelliteDllsProjectOutputGroupOutput);
+                            @(BuiltProjectOutputGroupOutput);
+                            @(_PathToPriFile)">
+        <PackagePath>runtimes/$(_runtimeOS)/lib/$(TargetFrameworkWithoutSuffix)</PackagePath>
+      </TfmSpecificPackageFile>
+      <TfmSpecificPackageFile Include="$(XmlDocFileRoot)1033\$(TargetName).xml" Condition="Exists('$(XmlDocFileRoot)1033\$(TargetName).xml')">
```

Comment:
Don't place doc files next to RID-specific assemblies, no-one will ever use them since these assemblies are not consumed at compile time.